### ISSUE TITLE
[CONCEPT] How to play an uploaded audio file?

### ISSUE THREAD

**Issue by LePhenix47** [LePhenix47/Audio-Visualizer_Younes-Lahouiti#3]
# Here's how

First we must get the file that uploaded by the user:
```js
const inputFile = document.querySelector("input[type=file]")

inputFile.addEventListener("change", (e)=>{
//We get the first and only file
const files = e.target.files[0];

})
```

**Comment by LePhenix47** [comment]
## 6) [Base₆₄ string](https://developer.mozilla.org/en-US/docs/Glossary/Base64) vs [Blobs](https://developer.mozilla.org/en-US/docs/Web/API/Blob)

There is another way to convert a file into readable data with the [`URL.createObjectUrl([File object])`](https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL_static), the only difference being that unlike the base₆₄ string, the URL generated is a temporary `Binary large object` URL and will be invalid once the user closes the tab 

In other words, the lifespan of a blob URL is tied to the document or web page that created it.
